contracts/0.4.24/interfaces/INodeOperatorsRegistry.sol

```diff
@@ -56,6 +56,17 @@ interface INodeOperatorsRegistry {
             uint64 usedSigningKeys
         );
 
+    function getNodeOperatorLimits(uint256 _nodeOperatorId)
```

Comment:
I propose to move this to the Staking Module interface. I believe these limits should apply to all modules